tests/vue-use.test.ts

```diff
@@ -148,6 +148,26 @@ describe("Vue.use", () => {
     expect(gtmPlugin.dataLayer()).toEqual(window["dataLayer"]);
   });
 
+  test("should allow dataLayer to be called with no event, without Typescript error", () => {
+    appendAppDivToBody();
+    const { app } = createAppWithComponent();
+    app.use(createGtm({ id: "GTM-DEMO" })).mount("#app");
+
+    const gtmPlugin: VueGtmPlugin = app.config.globalProperties.$gtm;
+
+    const dataLayer =  gtmPlugin.dataLayer()
+    if (dataLayer)
+      dataLayer.push({'user-id': 'user-123'})
```

Comment:
Could you please run `yarn format` and commit? :slightly_smiling_face:  
Sorry that I don't have an automation for that